.github/workflows/tag.yml

```diff
@@ -0,0 +1,24 @@
+name: RELEASE
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
```

Comment:
So how does this work? What does this buy us?

Reply:
when you push a tag a GitHub bot will and create the release on the releases page. This will help keep consistency on the release page when there are more people doing releases.